docs/Getting Started/Recurring Tasks.md

```diff
@@ -12,9 +12,12 @@ For example: `🔁 every weekday` means the task will repeat every week on Monda
 Every recurrence rule has to start with the word `every`.
 
 When you toggle the status of a recurring task to anything but "todo" (i.e. "done"), the original task that you wanted to toggle will be marked as done and get the done date appended to it, like any other task.
-In addition, _a new task will be put one line above the original task._
+In addition, _a new task will be put one line above the original task_ by default. You may change this behavior and make the new task appear on the line below the original task by enabling the `Next recurrence appears on the line below` setting.
```

Comment:
There is now a settings page in the docs that links to all the settings sections in the docs.

I would like to see the same sort of pattern here.

So that there is an explicit heading in the recurrence docs about this new setting. With an initial screenshot. And that the settings docs page links to this page, in the correct order.

Reply:
oh, I didn't know that, thanks for pointing out. Updated.